webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -8357,10 +8357,26 @@ var userOverride = {
                 ) {
                     floatWhere = 'mck-msg-center';
                 }
-                msgBoxColorStyle =
-                    floatWhere === 'mck-msg-right'
-                        ? 'km-custom-widget-background-color'
-                        : 'km-custom-widget-background-color-secondary';
+                if (floatWhere === 'mck-msg-right') {
+                    msgBoxColorStyle = 'km-custom-widget-background-color';
+                    if (kmAttchMsg === 'km-attach-msg') {
+                        messageClass = 'vis km-attach-msg-right';
+                        attachmentBox = 'km-attach-msg-right';
+                        if (!msg.message) {
+                            messageClass = 'n-vis';
+                        }
+                    }
+                } else {
+                    msgBoxColorStyle =
+                        'km-custom-widget-background-color-secondary';
+                    if (kmAttchMsg === 'km-attach-msg') {
```

Comment:
code inside the if else looks same, figure out a better way to implement it with minimal code and without repeating